package.json

```diff
@@ -27,11 +27,12 @@
     "github-from-package": "0.0.0",
     "minimist": "^1.1.2",
     "mkdirp": "^0.5.1",
-    "node-abi": "^2.0.0",
+    "node-abi": "git+https://github.com/saboya/node-abi.git#node-webkit",
```

Comment:
This should be a semver version, so it needs to be updated before merge.

Reply:
As soon as the PR is accepted by `node-abi`folks, I'll push a fixed version.

Reply:
@saboya Feel free to update to e.g. `^2.2.0`